lib/tasks/test_vmdb.rake

```diff
@@ -1,4 +1,5 @@
 require_relative "./evm_test_helper"
+require 'parallel_tests/tasks'
```

Comment:
@agrare I think we need a guard clause or rescue LoadError here since non dev/test setups won't have it.  I'm seeing this blow up doing rpm_build on master:

```
LoadError: cannot load such file -- parallel_tests/tasks
/root/BUILD/manageiq-gemset-18.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
/root/BUILD/manageiq-gemset-18.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
/root/BUILD/manageiq-gemset-18.0.0/gems/zeitwerk-2.6.12/lib/zeitwerk/kernel.rb:38:in `require'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:332:in `block in require'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:299:in `load_dependency'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:332:in `require'
/root/BUILD/manageiq/lib/tasks/test_vmdb.rake:2:in `<main>'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:326:in `load'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:326:in `block in load'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:299:in `load_dependency'
/root/BUILD/manageiq-gemset-18.0.0/gems/activesupport-6.1.7.6/lib/active_support/dependencies.rb:326:in `load'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/engine.rb:675:in `block in run_tasks_blocks'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/engine.rb:675:in `each'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/engine.rb:675:in `run_tasks_blocks'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/application.rb:528:in `run_tasks_blocks'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/engine.rb:464:in `load_tasks'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/railtie.rb:207:in `public_send'
/root/BUILD/manageiq-gemset-18.0.0/gems/railties-6.1.7.6/lib/rails/railtie.rb:207:in `method_missing'
/root/BUILD/manageiq/Rakefile:9:in `<top (required)>'
/root/BUILD/manageiq-gemset-18.0.0/gems/rake-13.1.0/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)

 ---> rake update:ui
```